src/webui/www/private/upload.html

```diff
@@ -8,6 +8,15 @@
     <link rel="stylesheet" href="css/Window.css?v=${CACHEID}" type="text/css" />
     <script src="scripts/lib/MooTools-Core-1.6.0-compat-compressed.js"></script>
     <script src="scripts/download.js?v=${CACHEID}"></script>
+    <script>
+        'use strict';
+
+        window.addEvent('domready', function() {
+            if (Browser.platform == 'ios') {
+                $('fileselect').accept = "";
```

Comment:
what about this:
```suggestion
                $('fileselect').accept = ".torrent";
```
will it work with ios?

Reply:
>... will it work with ios?

If not, then:
```suggestion
                $('fileselect').removeAttribute('accept');
```

Reply:
I used = ".torrent". 
IOS ignores it since it file extensions aren't used for the accept attribute. But if that changes we won't have to update it.